waltz-ng/client/applications/app-view.html

```diff
@@ -281,9 +281,9 @@ <h4 ng-bind="dataType | toDisplayName:'dataType'">
     </waltz-entity-statistic-section>
 
     <!-- surveys -->
-    <waltz-survey-instance-list survey-runs="ctrl.surveyRuns"
-                                survey-instances="ctrl.surveyInstances">
-    </waltz-survey-instance-list>
+    <waltz-survey-section survey-runs="ctrl.surveyRuns"
+                          survey-instances="ctrl.surveyInstances">
+    </waltz-survey-section>
 
```

Comment:
took opportunity to bring name of component in line with other 'section' level components.